### ISSUE TITLE
Accented characters are read incorrectly in WebViews

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#154]
```
What steps will reproduce the problem?
1. Open the E-mail application.
2. Find an email containing non ASCII characters ( try with chars é ë ê î 
ô  ç and so on)
3. Talkback will not pronounce them as they should be.

What is the expected output? What do you see instead?
The characters seem to be UTF-8 encoded, because Talkback says some Copyright 
sign and so on.
A sollution would be to manually UTF-8 decode the character to make talkback 
and the associated synthesizer pronounce them properly.



What version of the product are you using? On what operating system?
Using Android 4.2, the problem occurs with talkback 3.2.0 and 3.2.1 beta2.

Please provide any additional information below.

I also tested with several TTS engines (Acapela TTS, Google TTS engine, SVOX, 
Ivona, …), but the bug is still there.
Note that the problem doesn't occur in other types of views, even in GMail or 
Chrome which seems to use a "more recent" implementation of a webview (with all 
new features from ChromeVox).
```

Original issue reported on code.google.com by `plassiar...@gmail.com` on 28 Nov 2012 at 10:08

**Comment by GoogleCodeExporter** [comment]
Original comment by `alanv@google.com` on 7 Dec 2012 at 6:20
- Changed title: **Accented characters are read incorrectly in WebViews**

**Comment by GoogleCodeExporter** [comment]
```
This is an issue with WebView, and can't be fixed from TalkBack.
```

Original comment by `alanv@google.com` on 5 Mar 2013 at 8:00
- Changed state: **WontFix**
